src/ember-cli.ts

```diff
@@ -75,6 +75,8 @@ export class EmberCliManager {
                 return;
             };
 
+            result = fileNameSanitation(result);
```

Comment:
Is this doing anything? Seems like you planned to dasherize the string, but it is commented out, so I guess this is doing nothing?  
Also I am not sure if the dasherization makes sense since it is some hidden behaviour normal users don't know exist.